byron/ledger/executable-spec/src/Ledger/Update/Generators.hs

```diff
@@ -0,0 +1,74 @@
+-- |
```

Comment:
Were you about to put a module description here?

Reply:
Ups, auto-generated. I added a small comment.